sdk/maps/maps-common/review/maps-common.api.md

```diff
@@ -0,0 +1,187 @@
+## API Report File for "@azure/maps-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignalLike } from '@azure/abort-controller';
+import { CancelOnProgress } from '@azure/core-lro';
+import { KeyCredential } from '@azure/core-auth';
+import { PipelinePolicy } from '@azure/core-rest-pipeline';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
+
+// @public
+export interface BatchPoller<TBatchResult> extends PollerLike<PollOperationState<TBatchResult>, TBatchResult> {
+}
+
+// @public
+export class BatchPollerProxy<TBatchResult, TInternalBatchResult> implements BatchPoller<TBatchResult> {
+    constructor(internalPoller: PollerLike<PollOperationState<TInternalBatchResult>, TInternalBatchResult>, mapper: (res: TInternalBatchResult) => TBatchResult);
+    // (undocumented)
+    cancelOperation(options?: {
+        abortSignal?: AbortSignalLike;
+    }): Promise<void>;
+    // (undocumented)
+    getOperationState(): PollOperationState<TBatchResult>;
+    // (undocumented)
+    getResult(): TBatchResult | undefined;
+    // (undocumented)
+    isDone(): boolean;
+    // (undocumented)
+    isStopped(): boolean;
+    // (undocumented)
+    onProgress(callback: (state: PollOperationState<TBatchResult>) => void): CancelOnProgress;
+    // (undocumented)
+    poll(options?: {
+        abortSignal?: AbortSignalLike;
+    }): Promise<void>;
+    // (undocumented)
+    pollUntilDone(): Promise<TBatchResult>;
+    // (undocumented)
+    stopPolling(): void;
+    // (undocumented)
+    toString(): string;
+}
+
+// @public
+export type BBox = BBox2D | BBox3D;
+
+// @public
+export type BBox2D = [
+southWestLongitude: number,
+southWestLatitude: number,
+northEastLongitude: number,
+northEastLatitude: number
+];
+
+// @public
+export type BBox3D = [
+southWestLongitude: number,
+southWestLatitude: number,
+southWestElevation: number,
+northEastLongitude: number,
+northEastLatitude: number,
+northEastElevation: number
+];
+
+// @public
+export interface BoundingBox {
+    bottomRight: LatLon;
+    topLeft: LatLon;
+}
+
+// @public
+export function createAzureMapsKeyCredentialPolicy(azureKeyCredential: KeyCredential): PipelinePolicy;
+
+// @public
+export function createMapsClientIdPolicy(mapsClientId: string): PipelinePolicy;
+
+// @public
+export interface GeoJsonFeature extends GeoJsonObject {
+    // (undocumented)
```

Comment:
make sure to document all these undocumented properties

Reply:
I've added the reference and summary from the GeoJSON spec to the related interface, and I believe the information should be enough. The rest of the undocumented properties is considered unnecessary from our side.

For the `BatchPollerProxy`, since it's for internal so I leave it undocumented. Please refer to the commit: fd7722faa0810e9b120c0e2dd522813dcb30afa9